### PR TITLE
Use biome to format code

### LINKED ISSUE #44: add biome formatter check
the codebase is not formatted now, use biome to do format check in ci

### PR DESCRIPTION
**Pull request by xc2**
closes #44

Requesting a **fast forward merge**.

### CODE REVIEW

**Comment by xc2** [comment]
cc @hardfist @chenjiahan

File: biome.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  },
+  "files": {
+    "include": ["**/*.js", "**/*.ts", "**/*.json", "**/*.jsonc", "**/*.jsx", "**/*.tsx"],
+    "ignoreUnknown": true,
+    "ignore": ["**/auto-imports.d.ts", "**/components.d.ts"]
+  },
+  "javascript": {
+    "formatter": {}
+  },
+  "formatter": {
```
Comment (chenjiahan) [inline biome.json:20]: How about enable `"quoteStyle": "single"`? I think more frontend projects are using single quote

Comment (xc2) [reply biome.json:22]: Single quoteStyle adopted.

Half of the original codebase uses double and half uses single, so i adopted biome's default option before.

**Review by chenjiahan — APPROVED** [review]

**Comment by chenjiahan** [comment]
🚀🚀

**Pull request merged**